docs/maintain/databases.rst

```diff
@@ -167,3 +167,56 @@ To get the table related to a ``pg_toast_*`` table, take the number after ``pg_t
 .. code-block:: sql
 
    SELECT '16712'::regclass;
+
+
+
+Recovering the replica
+----------------------
+
+In the case that replication breaks or the replica server goes offline these are the steps to recover.
+There are two parts to this, mitigating the downtime and fixing replication.
+
+Mitigating downtime:
+
+# Enable public access to postgres on kingfisher-process1
+
+   Modify ``pillar/kingfisher.sls`` setting the following pillar value
+
+   .. code-block:: yaml
+
+      postgres:
+        public_access: True
+
+# Deploy changes to kingfisher-process1
+
+# Update DNS
+
+   Log into GoDaddy
+
+   Point the postgres-readonly DNS endpoint at kingfisher-process1
+
+   Wait for the DNS updates to complete. This will depend on the records TTL value.
+
+Done.
```

Comment:
Use numbered lists for all individual instruction steps, not paragraphs. No need to say "done". The end of a list means "done".

```suggestion
#. Update DNS records:

   #. Login to `GoDaddy <https://sso.godaddy.com>`__
   #. If access was delegated, open `Delegate Access <https://account.godaddy.com/access>`__ and click the *Access Now* button
   #. Open `DNS Management <https://dcc.godaddy.com/manage/OPEN-CONTRACTING.ORG/dns>`__ for open-contracting.org
   #. Update the replica's CNAME record to point to the main server's A record: for example, point ``postgres-readonly`` to ``kingfisher-process1``
   #. Wait for the changes to propagate, which depends on the original TTL value
```

Reply:
Fixed